doc/api/vm.md

```diff
@@ -866,6 +866,36 @@ local scope, so the value `localVar` is changed. In this way
 `vm.runInThisContext()` is much like an [indirect `eval()` call][], e.g.
 `(0,eval)('code')`.
 
+## vm.compileFunction
+<!-- YAML
+added: REPLACEME
+-->
+* `code` {string} The JavaScript code to compile and run.
+* `params` {Array|undefined} An array of string containing all paramters for
+  the function.
+* `options` {Object}
+  * `filename` {string} Specifies the filename used in stack traces produced
+    by this script.
+    **Default:** `''`
```

Comment:
1. Missing period after `` **Default:**... `` here and in other items.
2. Unneeded line breaks: it seems all the `` **Default:**... `` parts in this section can be placed on the previous lines to save some space.